src/GuiBase/BaseApplication.cpp

```diff
@@ -215,6 +216,12 @@ BaseApplication::BaseApplication( int argc,
     m_engine->initialize();
     addBasicShaders();
 
+    // Register the GeometrySystem converting loaded assets to meshes
+    m_engine->registerSystem( "GeometrySystem", new Ra::Engine::GeometrySystem, 1000 );
+    // Register the TimeSystem managing time dependant systems
+    Scalar dt = ( m_targetFPS == 0 ? 1_ra / 60_ra : 1_ra / m_targetFPS );
+    m_engine->registerSystem( "TimeSystem", new Ra::Engine::TimeSystem( dt, 300 * dt ), 1000 );
```

Comment:
Why 300*dt and 1000 ??

Reply:
Since for now it must have a max, I just put 300 frames, which seems reasonable.